src/client/Microsoft.Identity.Client/Internal/ClaimsHelper.cs

```diff
@@ -28,7 +28,7 @@ internal static class ClaimsHelper
             return claims;
         }
 
-        private static JObject MergeClaimsIntoCapabilityJson(string claims, JObject capabilitiesJson)
+        internal static JObject MergeClaimsIntoCapabilityJson(string claims, JObject capabilitiesJson)
```

Comment:
Changed as internal to use from the KerberosSupplementalTicketManager.AddKerberosTicketClaim() to merge the Kerberos Supplemental Ticket claims into the token request claims body parameter.